game-core/src/main/java/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -415,7 +416,7 @@ private MouseDetails convert(final MouseEvent me) {
   }
 
   private boolean unitsChanged(final Tuple<Territory, List<Unit>> newUnits) {
```

Comment:
This method implementation is effectively the inverse of an `equals()` method for `Tuple`.  I would have just replaced it with `!Objects.equals(currentUnits, newUnits)`, but it uses a different comparison for the second tuple element than is used in `Tuple#equals()`.  `CollectionUtils#equals()` checks if its arguments are _equivalent_ not necessarily _equal_.  Therefore, if I used `Objects#equals()`, it would almost assuredly change the runtime behavior and possibly introduce a regression.